django_fastdev/apps.py

```diff
@@ -83,6 +83,15 @@ def get_gitignore_path():
         return None
 
 
+def get_venv_path():
+    return os.environ.get("VIRTUAL_ENV", None)
```

Comment:
I understand this PR is moving this code around instead of adding it, but I wanted to point out that this usage of `VIRTUAL_ENV` as an env var is probably unreliable (see #39 and #36).

Reply:
good point! I did a little bit of research to reliably determine the best ways to get the venv path, and chained them all into the get_venv_path() func now. should be hard to find a scenario where it won't pick up now I think